families.json

```diff
@@ -378,15 +378,15 @@
   {"family": "Kadwa", "tags": []},
   {"family": "Kalam", "tags": ["handwritten"]},
   {"family": "Kameron", "tags": ["slab", "egyptian"]},
-  {"family": "Kanit", "tags": ["humanist", "futuristic"]},
-  {"family": "Kantumruy", "tags": []},
+  {"family": "Kanit", "tags": ["humanist", "futuristic", "loopless Thai"]},
```

Comment:
@nsmsn can "loopless Thai" just be "loopless"? Or is there a strong distinction that you are making here?